app/posts/manage-teacher-training-applications/2021-02-25-filter-by-subject.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Filter applications by subject
+description: Let users filter the applications list by the course’s subject
+date: 2021-02-25
+---
+
+We introduced a subject filter to allow users to filter by the course’s subject in the applications list.
+
+## Why we are doing this
```

Comment:
Kill heading, but the text within this section can be the intro

Reply:
Updated as requested.